R-package/src/lightgbm_R.cpp

```diff
@@ -851,6 +867,72 @@ SEXP LGBM_BoosterPredictForMat_R(SEXP handle,
   R_API_END();
 }
 
+struct SparseOutputPointers {
+  void* indptr; int32_t* indices; void* data; int indptr_type; int data_type;
```

Comment:
```suggestion
  void* indptr;
  int32_t* indices;
  void* data;
  int indptr_type;
  int data_type;
```

Could you please break these declarations onto individual lines, to make it a bit easier to understand this object's structure when reading the code?